ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -118,18 +122,37 @@ - (NSDictionary *) getStorageDictionary {
 }
 
 - (NSString *) getSystemName {
+#if TARGET_OS_OSX
+    NSProcessInfo *processInfo = [NSProcessInfo processInfo];
+    log
```

Comment:
what is log? (I'm learning it all myself)

Reply:
Hm interesting, random string on my end but didn't break the build. Ill remove